model/config.go

```diff
@@ -2683,6 +2683,11 @@ func (s *PluginSettings) SetDefaults(ls LogSettings) {
 		s.PluginStates["com.mattermost.plugin-incident-management"] = &PluginState{Enable: true}
 	}
 
+	if s.PluginStates["com.mattermost.plugin-channel-export"] == nil {
+		// Enable the channel export plugin by default
+		s.PluginStates["com.mattermost.plugin-channel-export"] = &PluginState{Enable: true}
```

Comment:
I'm still wondering if we can mitigate the impact of not working slash command for TE users. Maybe we only enable the plugin for EE installations because TE installations won't have the required license file anyway.

Reply:
If we do this, it would have to be outside this code path -- possibly at runtime of the plugin itself? But that could get tricky if a license isn't yet installed on a fresh EE instance but is about to be.

I see this as one of those annoying "enterprise nag" sort of things, where there's a button on display but it clearly communicates a need to upgrade first. Maybe running the plugin at all in this context should be on the chopping block, so it behaves like IM?

Deferring to @tao here, but I believe this is working as desired for TE users at present.

Reply:
> If we do this, it would have to be outside this code path

Why can't we check that in this code path? Just to be clear here: I'm suggesting to enable the plugin based on the installation type, not based on a configured license.

Reply:
Ah, TE vs. EE installations! Yes, that would validly that cut out a set of non-EE users, though it leaves the EE-but-TE users still affected. I think this is a solid improvement for both plugins, IMO.